src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/UpdateVCFSequenceDictionary.java

```diff
@@ -108,20 +108,24 @@ public void onTraversalStart() {
         getDefaultToolVCFHeaderLines().forEach(line -> outputHeader.addMetaDataLine(line));
         sourceDictionary = getBestAvailableSequenceDictionary();
 
-        // Warn and require opt-in via -replace if we're about to clobber a valid sequence
-        // dictionary. Check the input file directly via the header rather than using the
+        // If -replace is set, do not need to check the sequence dictionary for validity here -- it will still be
+        // checked in our normal sequence dictionary validation. Warn and require opt-in via -replace if we're about to
+        // clobber a valid sequence dictionary. Check the input file directly via the header rather than using the
         // engine, since it might dig one up from an index.
-        SAMSequenceDictionary oldDictionary =
-                inputHeader == null ? null : inputHeader.getSequenceDictionary();
-        if ( (oldDictionary != null && !oldDictionary.getSequences().isEmpty()) && !replace) {
-            throw new CommandLineException.BadArgumentValue(
-                    String.format(
-                            "The input variant file %s already contains a sequence dictionary. " +
-                            "Use %s to force the dictionary to be replaced.",
-                            getDrivingVariantsFeatureInput().getName(),
-                            REPLACE_ARGUMENT_NAME
-                    )
-            );
+        if (!replace) {
+            SAMSequenceDictionary oldDictionary =
+                    inputHeader == null ? null : inputHeader.getSequenceDictionary();
+            if (oldDictionary != null && !oldDictionary.getSequences().isEmpty())  {
+                throw new CommandLineException.BadArgumentValue(
+                        String.format(
+                                "The input variant file %s already contains a sequence dictionary. " +
+                                        "Use %s to force the dictionary to be replaced.",
+                                getDrivingVariantsFeatureInput().getName(),
+                                REPLACE_ARGUMENT_NAME
+                        )
+                );
+            }
+
```

Comment:
Can you update the doc for this tool (second paragraph) to include the --disable-sequence-dictionary-validation arg:

>  <p>This tool is designed to update the sequence dictionary in a variant file using a dictionary from another variant,
>   alignment, dictionary, or reference file.  If the sequence dictionary in the input VCF is being replaced because it
>   isn't valid, use --disable-sequence-dictionary-validation to suppress validation of the old dictionary. The replacement
>   dictionary must be valid, i.e. it must contain a sequence record for all variants in the target file. The dictionary
>   lines start with '##contig='.
>   </p>